src/style.scss

```diff
@@ -4,6 +4,14 @@
 @tailwind components;
 @tailwind utilities;
 
+[data-color-scheme='dark'] {
+  color-scheme: dark;
+}
+
+[data-color-scheme='light'] {
+  color-scheme: light;
+}
+
 :root,
 .light-mode {
   --primary-color: #384aff;
```

Comment:
Can't we just add `color-scheme: light;` here?

Reply:
No, couldn't get that to work.

Reply:
I also don't see any better way. If the dark mode shall be applied to the main browser window's scrollbar, the root element (`<html>`) needs some class/attribute, which requires this custom piece of code in the watcher.

Just one thing: With this we don't need `.light-mode` and `.dark-mode` on the app div.

Reply:
> Just one thing: With this change, `.light-mode` and `.dark-mode` on the app div are redundant now. Maybe this could be cleaned up before merge.

Are they really, I'm not sure I understand you but these variables are still used.

Reply:
Well.... I just see this:

![image](https://user-images.githubusercontent.com/6792578/147159501-e1355512-7193-4327-8660-e4337194abdd.png)

Reply:
`[data-color-scheme='light'] #app` is the same as `.light-mode`. That's what I mean.